### DISCUSSION TITLE
Change of license

### DISCUSSION THREAD

**Discussion by 1nv1** [neutralinojs/neutralinojs#423] [General]
Hi! Some days ago I found neutralinojs and I think is a great project. But, I noticed yesterday that you change license from MIT to GPL3. Why? I looked the issues section and you discussed about change GPL3 to MIT. Do you like change the license for compatibility issues?
Regards

**Comment by shalithasuranga** [comment]
Hey.. Thanks for checking Neutralinojs. The reason is that our HTTP server implementation on linux/mac uses GPL-v3.0 licensed code. By mistake, we used MIT initially. But community pointed out some licensing issues: https://github.com/neutralinojs/neutralinojs/issues/226
According to the GPL-v3.0, we cannot use another license, if we use GPL-v3.0 code. Therefore, I've just updated the license.

**Reply by 1nv1** [reply]
Thanks for you reply. Have a great day!

**Reply by shalithasuranga** [reply]
You too @1nv1
